DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Dashboard/TopCoursesController.cs

```diff
@@ -11,12 +11,12 @@
     [Route("/TrackingSystem/Centre/TopCourses")]
     public class TopCoursesController : Controller
     {
-        private readonly ICourseService courseService;
-        private const int TopNumberOfCourses = 10;
+        private readonly ICourseService CourseService;
+        private const int NumberOfTopCourses = 10;

```

Comment:
Is this naming change intentional?

Reply:
Sorry, highlighted the wrong line. The CourseService

Reply:
With the configuration -> Configuration change on the view, I changed my resharper settings so that it would be preferred as uppercase rather than lowercase (Code editing -> C# -> Naming style -> Instance Fields (private) = UpperCamelCase rather than lowerCamelCase). This then suggested that courseService should be CourseService. Pretty sure that most other files file lowerCamelCase for these private fields, so I'm not sure why we are only seeing this issue now.

Reply:
Yeah the setting for at least us (and maybe everyone else except Stella?) says that it should be courseService & configuration.
Whereas Stella was getting Fields (not private) errors with lower case configuration

Reply:
Well I'll just stick with what we've got for other files then. Reverted CourseService back to courseService. Kept Configuration as Configuration.